### ISSUE TITLE
Ingame chat time stamps should show ingame time

### ISSUE THREAD

**Issue by PunkPun** [OpenRA/OpenRA#20146]
Inside the In-game menu chat (reworked in #19458) times stamps should point not to current time but to the ingame time

**Comment by PunkPun** [comment]
Note for anyone implementing it:
In-lobby - real world time timestamps
In-game - in-game timer (currently real world timestamp)
In-replays - set lobby timestamps to 00:00 or change formatting to + -, and for in-game use in-game timer (currently timestamp not displayed)

**Comment by anvilvapre** [comment]
Wouldn't it be better to just display game time as world tick integer. And display actual calender time along with world tick index everywhere together. 

I.e. 00:00 format for game/world time doesn't make sense if the unit is not seconds. Or rather scale all displayed time to gamespeed - so values are actual seconds again. I.e. the build time for a barrack would be x seconds over y ticks.

**Comment by dragunoff** [comment]
Ticks are irrelevant when we talk about game time - i.e. a game that takes 5 minutes does so no matter what the game speed is.

**Comment by anvilvapre** [comment]
So the tooltip in the production tab shows the build time of units in real seconds? So i.e. the tooltip for a barrack shows a different build time when the game is played at normal speed compared to speed fastest?

**Comment by dragunoff** [comment]
> So the tooltip in the production tab shows the build time of units in real seconds? So i.e. the tooltip for a barrack shows a different build time when the game is played at normal speed compared to speed fastest?

Yes, it shows the build time in real-world seconds so it's different between slowest, normal, fastest, etc
